ministryofjustice/correspondence_tool_staff
# CT-3343 Update postgres version to 12.3

## Description
This should only be merged when all of the namespaces are on Postgres 12.3

## Self-review checklist




### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-3343

### Deployment
Build and deploy manually - to each namespace after upgrading postgres

### Manual testing instructions
Regression testing, but log into pods and check version of postgres client by typing psql -V
